packages/foundry/src/create.ts

```diff
@@ -0,0 +1,106 @@
+import path from "node:path";
+import { type OptionValues } from "@commander-js/extra-typings";
+
+import { cloneProjectTemplate } from "./cloneProjectTemplate.js";
+import { replaceTokens } from "./replaceTokens.js";
+
+const BaseRepositoryAddress = "Workleap/wl-foundry-cli/templates";
+
+type TemplateId = "host-application" | "remote-module" | "static-module";
+
+interface TemplateInterface {
```

Comment:
TemplateDetails, we try to avoid prefixing with I, or adding Interface to the name